src/pages/Strategies/Strategies.tsx

```diff
@@ -18,14 +26,35 @@ const Strategies = (): JSX.Element => {
           transaction fees!
         </P>
       </Card>
-      <StyledList>
-        {Object.values(StrategyType).map((strategyType) => (
-          <StrategyCard key={strategyType} strategyType={strategyType} />
-        ))}
-        <Card alignItems='center' justifyContent='center'>
-          <P size='xs'>More Strategies coming soon</P>
-        </Card>
-      </StyledList>
+      {strategies ? (
+        <StyledList>
+          {strategies.map((strategy) => {
+            const { title, summary } = getContent(strategy, t);
+
+            const to = PAGES.STRATEGY.replace(`:${URL_PARAMETERS.STRATEGY.TYPE}`, strategy.type);
+
+            return (
+              <Link key={strategy.type} to={to}>
+                <StrategyCard
+                  risk={strategy.risk}
+                  ticker={strategy.currency.ticker}
+                  interestRate={strategy.interestRate}
+                  title={title}
+                  description={summary}
+                />
+              </Link>
+            );
+          })}
+          <Card alignItems='center' justifyContent='center'>
+            <P size='xs'>More Strategies coming soon</P>
+            <TextLink size='xs' underlined to={'#'}>
+              Request strategies
+            </TextLink>
+          </Card>
+        </StyledList>
+      ) : (
+        <FullLoadingSpinner />
```

Comment:
We were missing loading screen. Its okay to load here the data and then the user wont have to wait for this data on the next screen. At some point we will bring skeletons.